index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] There is a big difference between your design for the featured film section and the original design Please make this section similar to the original design cause you need to stick it as much as possible :+1: 
![Screenshot from 2022-07-28 02-24-52](https://user-images.githubusercontent.com/6615394/181406919-da51bdab-31ff-4341-bd31-ccb0f57d92e2.png)
![Screenshot from 2022-07-28 02-23-04](https://user-images.githubusercontent.com/6615394/181406924-b1bfe4d6-b1aa-4799-9b7b-24f2bf872dac.png)